ocamldoc/odoc_html.ml

```diff
@@ -36,6 +36,12 @@ let charset = ref "iso-8859-1"
 (** The functions used for naming files and html marks.*)
 module Naming =
   struct
+    (** The prefix for modules marks. *)
+    let mark_module = "MODULE_"
```

Comment:
The other marks don't seem to include the `_` character, should we follow the same style for the new ones?

Reply:
Yes, complicating the logic here was a bad idea. Fixed, thanks.